api/db/create_db.sql

```diff
@@ -22,6 +22,9 @@ GRANT SELECT, INSERT, UPDATE, DELETE, CREATE TEMPORARY TABLES ON `synth_c_%`.* T
 GRANT SELECT, INSERT, UPDATE, DELETE, CREATE TEMPORARY TABLES ON `sr_%`.* TO '${WORKBENCH_DB_USER}'@'%';
 GRANT SELECT, INSERT, UPDATE, DELETE, CREATE TEMPORARY TABLES ON `sc_%`.* TO '${WORKBENCH_DB_USER}'@'%';
 
+-- for tanagra_db workbench user performs both liquibase changes and services api-tanagra-db calls
```

Comment:
This has already by committed in a prior PR. You need to rebase this branch.

Reply:
:-)